docs/reference/search/suggesters/phrase-suggest.asciidoc

```diff
@@ -17,36 +17,94 @@ co-occurrence and frequencies.
 
 ==== API Example
 
-The `phrase` request is defined along side the query part in the json
-request:
+In general the `phrase` suggester requires special mapping up front to work.
+The `phrase` suggester examples on this page need the following mapping to
+work. The `reverse` analyzer is used only in the last example.
 
 [source,js]
 --------------------------------------------------
-curl -XPOST 'localhost:9200/_search' -d '{
-  "suggest" : {
-    "text" : "Xor the Got-Jewel",
-    "simple_phrase" : {
-      "phrase" : {
-        "analyzer" : "body",
-        "field" : "bigram",
-        "size" : 1,
-        "real_word_error_likelihood" : 0.95,
-        "max_errors" : 0.5,
-        "gram_size" : 2,
-        "direct_generator" : [ {
-          "field" : "body",
-          "suggest_mode" : "always",
-          "min_word_length" : 1
-        } ],
-        "highlight": {
-          "pre_tag": "<em>",
-          "post_tag": "</em>"
+POST test
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 1,
+      "analysis": {
+        "analyzer": {
+          "trigram": {
+            "type": "custom",
+            "tokenizer": "standard",
+            "filter": ["standard", "shingle"]
+          },
+          "reverse": {
+            "type": "custom",
+            "tokenizer": "standard",
+            "filter": ["standard", "reverse"]
+          }
+        },
+        "filter": {
+          "shingle": {
+            "type": "shingle",
+            "min_shingle_size": 2,
+            "max_shingle_size": 3
+          }
         }
       }
     }
+  },
+  "mappings": {
+    "test": {
+      "properties": {
+        "title": {
+          "type": "string",
+          "fields": {
+            "trigram": {
+              "type": "string",
+              "analyzer": "trigram"
+            },
+            "reverse": {
+              "type": "string",
```

Comment:
This *is* master. Man, I wish the docs failed when I used deprecated syntax! I want #17512!